trl/trainer/ppo_trainer.py

```diff
@@ -553,13 +553,12 @@ def batched_forward_pass(
         # automatically padded by the datacollator to indicate padding tokens
         if self.is_encoder_decoder:
             input_data = self.data_collator(
-                [{"input_ids": q, "attention_mask": torch.ones_like(q).to(self.accelerator.device)} for q in queries]
-            )
+                [{"input_ids": q, "attention_mask": torch.ones_like(q)} for q in queries]
+            ).to(self.accelerator.device)
 
-            attention_mask = [torch.ones_like(r) for r in responses]
```

Comment:
Hum why this has been removed? 🤔

Reply:
because seems attention_mask variable is never used

Reply:
Perfect, can you run the styling checks so that the testing suite will be executed?

Reply:
just did